src/main/java/org/commcare/formplayer/application/MenuController.java

```diff
@@ -63,6 +64,19 @@ public class MenuController extends AbstractBaseController {
 
     private final Log log = LogFactory.getLog(MenuController.class);
 
+    @RequestMapping(value = Constants.URL_INTERVAL_SYNC_DB, method = RequestMethod.POST)
+    @UserLock
+    @UserRestore
+    @AppInstall
+    public SyncDbResponseBean scheduleSync(@RequestBody SessionNavigationBean sessionNavigationBean,
```

Comment:
I think this should go in the `UtilController` with the other sync endpoints.

Reply:
[d0bc87f](https://github.com/dimagi/formplayer/pull/1544/commits/d0bc87f202c75b6d8de845cbe9c45eaf6a92eace)